tests/plugins/pastefromwordimage/manual/pastewordwithimage.html

```diff
@@ -0,0 +1,4 @@
+<textarea id="editor" cols="30" rows="10"></textarea>
+<script>
```

Comment:
This test should be ignored in browsers not supporting Clipboard API IMHO.

Reply:
Alternatively you may describe what is the expected behaviour for such browsers.